bodhi/__init__.py

```diff
@@ -17,7 +17,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Fedora's update manager."""
 
-__version__ = "5.5"
+__version__ = "5.6"
```

Comment:
Maybe this should set to `5.6.0` as Kevin had to patch the version number for packaging Bodhi in Koji

Reply:
:+1: